### PR TITLE
IS A DEV REGISTRATION

### PR DESCRIPTION
**Pull request by RAYXSTUDIOZGAMES**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/RAYXSTUDIOZ-DEV.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "MY PORTFOLIO",
+  "repo": "https://github.com/RAYXSTUDIOZGAMES/PORTFOLIO/tree/main",
+  "owner": {
+    "username": "SYED-RAYAN-IS-A.DEV",
+    "email": "syedyaseeralirayan@gmail.com",
+    "twitter": "SYEDRAYAN168350"
+  },
+  "record": {
+    "CNAME": "rayxstudiozgames.github.io"
```
Comment (21Z) [inline domains/RAYXSTUDIOZ-DEV.json:10]: 404 Site not found

File: sussyrayan.json
Comment (21Z) [inline sussyrayan.json:1]: - Invalid JSON
- File should be in domains folder

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/RAYXSTUDIOZ-DEV.json (hunk 1 above)
Comment (Stef-00012) [reply domains/RAYXSTUDIOZ-DEV.json:10]: he has https://rayxstudiozgames.github.io/OCT23-SCHEDULE/ so it's valid

File: sussyrayan.json
Comment (Stef-00012) [inline sussyrayan.json:1]: as 21Z said, the file must be in the domains folder and you're missing a `{` in the JSON

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/RAYXSTUDIOZ-DEV.json
Comment (Stef-00012) [inline domains/RAYXSTUDIOZ-DEV.json:1]: file name should be lowercase

File: sussyrayan.json
Comment (Stef-00012) [inline sussyrayan.json:1]: you can only add files in the `domains` folder
also missing opening `{` in the json

File: sussyrayan.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+"owner": {
+           "username": "sussy rayan-is-good",
+           "email": "syedyaseeralirayan@gmail.com",
+           "discord": "1168249368252522576"
+        },
+
+        "record": {
+            "A": ["172.67.178.60"]
```
Comment (Stef-00012) [inline sussyrayan.json:8]: I cannot reach your website, error 1003

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**